### PR TITLE
Pytorch Component library maintenance

### PR DESCRIPTION
**Pull request by rabea-al**
# Description

- Updated README.md to maintain consistent formatting across all files.
- Locked package version in requirements.txt to prevent conflicts.
- Reloaded canvas to ensure all modifications are applied.
- Bug fixes

## Pull Request Type

- [ ] Xircuits Component Library Code
- [ ] Workflow Example
- [x] Documentation
- [x] Others (Please Specify) **lock requirements**

## Type of Change

- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Tests

- Check Readme and run the examples

## Tested on?

- [ ] Windows  
- [x] Linux fedora
- [ ] Centos 
- [ ] Mac  
- [ ] Others  (State here -> xxx )

### CODE REVIEW

File: torch_nn_layers.py [hunk 1]
```diff
@@ -91,12 +91,13 @@ class TorchAddLinearLayer(Component):
...
     bias: InArg[bool]
     model_out: OutArg[list]
-
+    
     def execute(self,ctx) -> None:
         bias = True if self.bias.value is None else False
         in_size = self.in_features.value
         out_size = self.out_features.value
-        
+        print("model_in value before adding Linear layer:", type(self.model_in.value))
```
Comment (treo) [inline torch_nn_layers.py]: looks like a left-over debug print?

Comment (rabea-al) [reply torch_nn_layers.py]: yes, there was a bug here and  I forgot remove it

Comment (rabea-al) [reply torch_nn_layers.py]: Done

**Review by MFA-X-AI — CHANGES_REQUESTED** [review]
Looks like there's a one line git conflict here. Gotta resolve that before merging.

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-torch
-torchvision
+torch==2.4.1
+torchvision=0.19.1
```
Comment (MFA-X-AI) [inline requirements.txt]: Typo here

**Review by MFA-X-AI — CHANGES_REQUESTED** [review]
The test is failing, there's a typo in the requirements.txt.
I believe you can run the test on your fork, right? Let me know if it doesn't or you need help setting it up.

File: requirements.txt (hunk 2 above)
Comment (rabea-al) [reply requirements.txt]: okay

**Pull request merged**